mcli/mcli-llama2-finetune.yaml

```diff
@@ -0,0 +1,151 @@
+integrations:
+- integration_type: git_repo
+  git_repo: mosaicml/llm-foundry
+  git_commit: 148c0793907a6afa48a892620e637ef5f90cdaf1 # TODO: repin this after next release
```

Comment:
I assume you have a reminder for this TODO

Reply:
haha jira it is